tests/testthat/test-derive_vars_transposed.R

```diff
@@ -90,7 +90,64 @@ test_that("ATC variables are merged properly", {
     "BP40257-1002", "19", "2791596", "SPIRONOLACTONE", "C", "C03", "C03D", "C03DA"
   )
   # nolint end
-  actual_output <- derive_vars_atc(cm, facm)
+  actual_output <- derive_vars_atc(
+    dataset = cm,
+    dataset_facm = facm,
+    id_vars = exprs(FAGRPID)
+  )
+
+  expect_dfs_equal(expected_output, actual_output, keys = c("USUBJID", "CMDECOD", "ATC4CD"))
+})
+
+
+
+
+
+test_that("ATC variables are merged properly", {
```

Comment:
Can you follow the numbering conventions in the test files please

Reply:
Test file is now udpated.